### PR TITLE
Web Applications and HTTP exercise updates

### PR DESCRIPTION
**Pull request by sohbaker**
Introduces the following changes: 
- updates the instructions in the README 
- fixes minor typos
- changes the example URL
- adds sample answers to exercise questions
- adds starter HTML file for HTML form exercise

### CODE REVIEW

File: exercises/web-applications/login-form.html
Comment (Genyus) [inline exercises/web-applications/login-form.html:1]: Shouldn't this have a basic HTML skeleton rather than being completely empty?

**Review by Genyus — CHANGES_REQUESTED** [review]

**Comment by tanyapowell** [comment]
Are these changes part of [TECH-427](https://codingblackfemales.atlassian.net/browse/TECH-427?atlOrigin=eyJpIjoiOGU2NmEwNGZlZGFiNDJhY2FkZTE5YjZmNjdhYTU0YTUiLCJwIjoiaiJ9)?

**Comment by Genyus** [comment]
@tanyapowell no, this PR isn't related to that ticket

File: exercises/web-applications/login-form.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login Form</title>
+</head>
+
+<body>
+    <!-- add content to your login form here -->
+    <p>Hello World!</p>
```
Comment (Genyus) [inline exercises/web-applications/login-form.html]: Could make the boilerplate content a little more relevant

```suggestion
    <h1>Login</h1>
    <!-- add content to your login form here -->
```

**Review by Genyus — CHANGES_REQUESTED** [review]
Minor change suggested to HTML boilerplate

**Review by Genyus — APPROVED** [review]
LGTM

**Pull request merged**